### PR TITLE
add Major Concepts and clean up

### PR DESCRIPTION
**Pull request by biancabuzea200**
-crete page "Major Concepts" to facilitate onboarding and present major relevant concepts for the LUKSO ecosystem. The page will be extended overt time.

### CODE REVIEW

File: docs/learn/concepts.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# Major Concepts
+
+## Universal Profile (UP):
+
+A Universal Profile (UP) is a smart contract-based account that acts as the center for all your blockchain interactions. UPs are generic for various use cases, such as regular wallets, and can interact directly with onchain applications. Additionally, they also feature permission and asset management, updatable profile information, and let you interact as one persona through different controller keys.
+
+## Controller:
+
+In the [LSP6 Key Manager](../standards/universal-profile/lsp6-key-manager.md), the term "controller" refers to a permissioned address. These addresses have permissions that allow them to perform certain actions on the [LSP0ERC725Account](../standards/universal-profile/lsp0-erc725account.md) linked with the Key Manager (e.g., setting data or transferring LYX from the account).
```
Comment (CJ42) [inline docs/learn/concepts.md:14]:
```suggestion
In the [LSP6 Key Manager](../standards/universal-profile/lsp6-key-manager.md), the term _"controller"_ refers to a permissioned address. These addresses have permissions that allow them to perform certain actions on the [LSP0ERC725Account](../standards/universal-profile/lsp0-erc725account.md) linked with the Key Manager (e.g., setting data or transferring LYX from the account).
```

File: docs/learn/concepts.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: '📑 Major Concepts'
```
Comment (CJ42) [inline docs/learn/concepts.md:2]: Maybe we could also use the term "Glossary" as well? It seems to fit the purpose of this page.

```suggestion
sidebar_label: '📑 Concepts & Glossary'
```

or 

```suggestion
sidebar_label: '📑 Major Concepts& Glossary'
```

File: docs/learn/concepts.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+sidebar_position: 2
+---
+
+# Major Concepts
+
+## Universal Profile (UP):
```
Comment (CJ42) [inline docs/learn/concepts.md:8]: Is the colon ":" needed at the end?

As it appears like that in the sidebar. I would suggest removing it, but this is just my personal preference and I am not too strong about it.

image

File: docs/learn/concepts.md (hunk 1 above)
Comment (CJ42) [reply docs/learn/concepts.md:14]: @biancabuzea200 This definition is also in the LSP6 page. Do you think it is necessary to have it in both places in this case?

image

File: docs/learn/concepts.md (hunk 3 above)
Comment (biancabuzea200) [reply docs/learn/concepts.md]: removed

File: docs/learn/concepts.md (hunk 1 above)
Comment (CJ42) [reply docs/learn/concepts.md]: Decided to keep duplication in both pages

**Review by CJ42 — APPROVED** [review]

**Pull request merged**